csunplugged/topics/content/en/binary-numbers/unit-plan/lessons/codes-for-letters-using-binary-representation-junior.md

```diff
@@ -0,0 +1,136 @@
+# Codes for letters using binary representation for junior students
+
+## Key questions
+
+How do you think a computer knows which letter to show on the screen?
+
+{panel type="teaching" title="Teaching observations"}
+
+Discussion may start at the 26 letters of the English alphabet, and then expand to other characters on the keyboard, including capital letters, digits and punctuation.
+Students may be aware that other languages can have thousands of characters, and the range of characters is also expanding as emoticons are invented!
+
+{panel end}
+
+## Lesson starter
+
+{panel type="general" title="Notes on resources"}
+
+There is also an online interactive version of the binary cards [here](http://www.csfieldguide.org.nz/en/interactives/binary-cards/index.html?digits=8), from the [Computer Science Field Guide](http://www.csfieldguide.org.nz/), but it is preferable to work with physical cards.
+
+{panel end}
+
+Can we match letters to numbers so that we can send coded messages to each other?
+
+How many letters are there in the alphabet?
+Let’s count them together on our alphabet cards.
+
+How can we represent the letters using numbers?
+(Guide students to the idea of using 1 for A, 2 for B, and so on.)
+
+We can represent numbers using binary, but in the last lesson with 4 bits, what was the biggest number we could represent? (15)
+
+How can we represent bigger numbers? (Add a card).
+How many dots on the extra card? (16)
+
+Give out the cards, and have the students place them on the table in the correct order (16, 8, 4, 2, 1).
+
+Now give them a number by saying "No, Yes, No, No, No" for the 5 cards. Ask them how many dots this produces.
+(The "Yes" is for the 8 card, so it's the number 8.)
+Which letter is number 8? ("H").
+This can be written on the board.
+
+Now give the next number, "No, Yes, No, No, Yes" (9).
+Which letter is number 9?
+("I", which can be written after the "H".)
+
+That's the whole message - "HI".
+
+Now try sending a different word to the class.
+The table below shows the binary patterns for the 26-letter alphabet; you can use yes/no for 1/0, but you could also use other ways of saying them, such as "on/off", "up/down", or even "one/zero".
```

Comment:
Jack please update to: The Binary to Alphabet resource [link please] shows .....